chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.css

```diff
@@ -490,7 +490,7 @@ img {
   }
 
 
-}
+}port
```

Comment:
Is this a typo?  If so, please remove `port` from css file.

Reply:
Removed typo.